build.gradle

```diff
@@ -175,7 +178,7 @@ project(':programming-core') {
                 'org.eclipse.jetty:jetty-servlet:9.2.29.v20191105',
                 'org.eclipse.jetty:jetty-xml:9.2.29.v20191105',
 
-                'org.tmatesoft.svnkit:trilead-ssh2:build213-svnkit-1.3-patch',
+                name: "trilead-ssh2-build-217-jenkins-activeeon-SNAPSHOT",
```

Comment:
why use a snapshot version and not the following ?

https://mvnrepository.com/artifact/org.jenkins-ci/trilead-ssh2/build-217-jenkins-27